.github/workflows/sprite.yml

```diff
@@ -0,0 +1,23 @@
+name: "Update sprite files"
+on:
+  schedule:
+  - cron: 0 0 * * *
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  sprite:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: technote-space/create-pr-action@v2
+      with:
+        EXECUTE_COMMANDS: |
+          git clone --depth 1 https://github.com/IsraelHikingMap/VectorMap.git
+          cp -up VectorMap/Icons/publish/sprite* IsraelHiking.Web/src/content/sprite
+          rm -fr VectorMap
+        COMMIT_MESSAGE: 'chore: update sprite files'
+        COMMIT_NAME: 'GitHub Actions'
+        COMMIT_EMAIL: 'example@example.com'
```

Comment:
Maybe we can use a different e-mail?

Reply:
As far as I understand, this e-mail is just a placeholder. The actual e-mails are sent from GitHub.
For example:

![image](https://user-images.githubusercontent.com/1304610/205240124-ac7738a4-ade0-4a41-857e-8e7d4081c4f3.png)